homeassistant/components/netgear/sensor.py

```diff
@@ -0,0 +1,105 @@
+"""Support for Netgear routers."""
+from dataclasses import dataclass
+import logging
+
+from homeassistant.components.sensor import (
+    DEVICE_CLASS_SIGNAL_STRENGTH,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE
+from homeassistant.core import callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .router import NetgearDeviceEntity, NetgearRouter, async_setup_netgear_entry
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class NetgearSensorDescription(SensorEntityDescription):
+    """Class that holds device specific info for a netgear sensor."""
+
+    attributes: tuple = ()
```

Comment:
The `attributes` attribute isn't used.

Reply:
I removed it, the whole NetgearSensorDescription was not needed, so I removed it.